frontend/mono_frontend/src/pages/JarsList/JarsList.jsx

```diff
@@ -0,0 +1,45 @@
+import { Link, useRouteLoaderData } from 'react-router-dom'
+import './JarsList.css'
+import { useEffect, useState } from 'react'
+import { BACKEND_URL } from '../../config/envs'
+import { JarIcon } from '../../components/JarIcon'
+import { ProgressBar } from '../../components/ProgressBar'
+import { JarsListView } from '../../components/JarsListView'
+
+export const getJars = async function (token) {
+	const endpoint = `${BACKEND_URL}/monobank/monojars`
+
+	const response = await fetch(endpoint, {
+		method: 'GET',
+		headers: {
+			'Content-Type': 'application/json',
+			Authorization: `Bearer ${token}`
+		}
+	})
+	const data = await response.json()
+
+	if (!response.ok) {
+		return null
+	}
+	return data
+}
+
+export const JarsList = () => {
+	const token = useRouteLoaderData('token')
+	const [jarsData, setJarsData] = useState(null)
+
+	useEffect(() => {
+		const fetchData = async function () {
+			try {
+				const access = await getJars(token)
+				setJarsData(access)
+			} catch (error) {
+				console.error('Error fetching card data:', error)
```

Comment:
suggestion: we have [PopUpManager](https://github.com/Iron-Cow/MonoProject/blob/master/frontend/mono_frontend/src/components/PopUpManager.jsx) and can use it to visualise some issues for user. 

Here is some [example](https://github.com/Iron-Cow/MonoProject/blob/a772f49d2b50359a64df66c02f8b48a5656f9ac0/frontend/mono_frontend/src/pages/LoginPage.jsx#L75)

Reply:
Done